docs/reference/connector/docs/release-notes/connectors-release-notes-8.16.0.asciidoc

```diff
@@ -0,0 +1,46 @@
+[[es-connectors-release-notes-8-16-0]]
+=== 8.16.0 connectors release notes
+
+[discrete]
+[[es-connectors-release-notes-8-16-0-breaking]]
+==== Breaking Changes
+
+* Directly accessing connector and sync job state through `.elastic-connectors*` indices is deprecated, and will be disallowed entirely in a future release. Instead, the Elasticsearch Connector APIs should be used. Connectors framework code now uses the Elasticsearch Connector APIs by default.
+See https://github.com/elastic/connectors/pull/2884[*PR 2902*]..
```

Comment:
I'm not sure about this one - from what I understand we haven't fully migrated to Connector APIs and some endpoints might be missing cc @jedrazb.

Should we not be too strong with our deprecation statement here?

Reply:
No, we should be strong with the deprecation notice. It is going away _eventually_ and that's what we're trying to communicate. Not that it _cannot_ be used in 8.16 (so @leemthompo IDK if this belongs under "Breaking Changes."), but that, starting now, customers need to start planning to move away from direct index access.